clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -158,16 +157,16 @@ public void testStartupAndTearDown() throws InterruptedException {
 
     @Test
     public void testRequestsTransferFromManagersToClientOnThreadRun() {
-        List<Optional<? extends RequestManager>> list = new ArrayList<>();
-        list.add(Optional.of(coordinatorRequestManager));
-        list.add(Optional.of(heartbeatRequestManager));
-        list.add(Optional.of(offsetsRequestManager));
+        List<RequestManager> list = new ArrayList<>();
```

Comment:
ditto

Reply:
```
List<RequestManager> list = List.of(coordinatorRequestManager, heartbeatRequestManager, offsetsRequestManager);
```

Reply:
Yes, fixed this as well. Thank you for your patience 😄